### PR TITLE
feat Weather widget

### PR DESCRIPTION
**Pull request by TejasShekar**
- Ignore the package upgrade files.
- Added weather widget (allow location for it to work)
- Made app basic responsive

### CODE REVIEW

**Review by pradipgharge — APPROVED** [review]
This is a small pr, looks good to merge from my end

File: src/App.css [hunk 1]
```diff
@@ -29,11 +32,30 @@ h2 {
   font-size: 3rem;
   background: transparent;
   border: none;
-  border-radius: 0;
+  border-radius: 0 !important;
```
Comment (xii-vi) [inline src/App.css:35]: Try to find another solution instead of important.

**Review by xii-vi — APPROVED** [review]
Code is clean.
Avoid to include package files.
Go through comments once before merging.

**Review by MaruthiPothuganti — COMMENTED** [review]
Well written and Clean code. Looks good to merge.

File: src/App.css (hunk 1 above)
Comment (TejasShekar) [reply src/App.css:35]: I had to overwrite styles coming from my component library.

**Pull request merged**